qa/rpc-tests/test_framework/authproxy.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, service_url, service_name=None, timeout=HTTP_TIMEOUT, connect
                                                   timeout)
         else:
             self.__conn = httplib.HTTPConnection(self.__url.hostname, port,
-                                                 False, timeout)
+                                                 timeout=timeout)
```

Comment:
Actually this was already fixed: https://github.com/jgarzik/python-bitcoinrpc/commit/7ebeebb4f61917fe590d980cb4f9aefdce2c8f25

But somehow we forgot to fetch it.